### ISSUE TITLE
Create data migration for setting correct timestamp on StoredMessage objects.

### ISSUE THREAD

**Issue by rwrzesien** [golemfactory/concent#690]
The already merged pull request https://github.com/golemfactory/concent/pull/641 is missing data migration which will set correct timestamp on StoredMessage objects. Timestamp can be retrieved from `StoredMessage.data` which is serialized Golem message.

**Comment by kbeker** [comment]
It might be ready solution:
```
StoredMessage = apps.get_model('core', 'StoredMessage')
for stored_message in StoredMessage.objects.all():
    stored_message.timestamp = parse_timestamp_to_utc_datetime(
        deserialize_message(stored_message).timestamp
    )
    stored_message.full_clean()
    stored_message.save()
```

**Comment by PaweuB** [comment]
Pivotal:
https://www.pivotaltracker.com/story/show/159431910

**Comment by Jakub89** [comment]
Closed via: #691
